### DISCUSSION TITLE
Magic DRC checker for GF180 MCUC

### DISCUSSION THREAD

**Discussion by spnadig** [RTimothyEdwards/magic#238] [Q&A]
Hi is the DRC checker on magic for GF180 MCUC up to date? Have folks successfully submitted designs to GF that is DRC clean from magic and was accepted by GF?

**Comment by spnadig** [comment]
@proppy

**Comment by RTimothyEdwards** [comment]
Development of the PDK was given over to Amro Tork, so I have not done much with developing the PDK for magic since then.  It is definitely not up to date.
